src/components/tl/dpu/tl_dpu.h

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_TL_DPU_H_
+#define UCC_TL_DPU_H_
+
+// #include "tl_dpu_coll.h"
+#include "components/tl/ucc_tl.h"
+#include "components/tl/ucc_tl_log.h"
+#include <ucp/api/ucp.h>
+
+#ifndef UCC_TL_DPU_DEFAULT_SCORE
+#define UCC_TL_DPU_DEFAULT_SCORE 20
+#endif
+
+#define UCC_TL_DPU_TEAM_CREATE_POLL 1000
+
+#define EXCHANGE_LENGTH_TAG 1ull
+#define EXCHANGE_RKEY_TAG 2ull
+#define EXCHANGE_ADDR_TAG 3ull
+
+typedef enum {
+    TL_DPU_UCP_REQUEST_ACTIVE,
+    TL_DPU_UCP_REQUEST_DONE,
+} tl_dpu_ucp_request_status_t;
+
+typedef struct tl_dpu_ucp_request {
```

Comment:
prefix for functions and structs "ucc_tl_dpu"

Reply:
fixed, and fixed in several other places